### DISCUSSION TITLE
따로 번역할 때 이 부분 레퍼런스로 참고해줘

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#306] [일반]
[위키 링크](https://github.com/RimWorldKorea/RMK/wiki/1.-%EB%B2%88%EC%97%AD%EC%9E%90-%EB%A7%A4%EB%89%B4%EC%96%BC#%E2%85%B1-rmk-%ED%98%91%EC%97%85-%EA%B8%B0%EC%B4%88-%EC%A7%80%EC%8B%9D)

@zilrty

**Comment by zilrty** [comment]
넹
